src/main/java/rssfeed/RssFeed.java

```diff
@@ -0,0 +1,34 @@
+package rssfeed;
+
+import org.junit.platform.commons.util.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class RssFeed {
+    public String title;
+    public String description;
+    public String link;
+    public String language;
+    public String copyright;
+
+    public ArrayList<RssItem> items = new ArrayList<RssItem>();
```

Comment:
```suggestion
    public List<RssItem> items = new ArrayList<>();
```
Prefer interfaces over concrete implementations